packages/template-typescript-minimal/app/components/_app/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {Provider} from 'slas-react-sdk'
```

Comment:
It is possible that an app is using multiple provider at a time. What do you think about using `SlasProvider` to make clearer?

Reply:
`SlasProvider` sounds sensible to me.

All naming stuff in this PR is open for feedback, but I'm looking for architectural feedback more at this moment.